### DISCUSSION TITLE
2022/03/29/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A82/

### DISCUSSION THREAD

**Discussion by giscus** [OOJ2003/OOJ2003.github.io#21] [Announcements]
# 2022/03/29/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A82/

更好的泛型单向链表（链栈）

https://ooj2003.github.io/2022/03/29/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A82/

**Comment by night-cruise** [comment]
Drop 这样实现更简洁：
```rust
while self.pop().is_some() {}
```
